### ISSUE TITLE
Scheduler and Notification : the scheduler accordian form should be opened only when create message form is filled up.  

### ISSUE THREAD

**Issue by deepika-Gopala** [OnePointGlobal/OnePoint-Global-Account-Toolkit#84]
step 2(scheduling) should be completed only after step 1(create message) is done.

**Comment by onepointpb** [comment]
This feature is not working because the Save button for Message tab stopped working after the below scenario.
Scenario:
----------
- Select Email server. 
- Enter Subject. Click 'Save'. The Save button will not work now.

At the end the feature is not letting the user schedule a survey.

**Comment by veereshms** [comment]
After saving the message scheduler is not getting displayed.

**Comment by premonepoint** [comment]
unable to reproduce

**Comment by onepointpb** [comment]
Keeping this issue on-hold. Cause we believe that because of session timeout or some sort of server issue this might have occurred. Different cases will be checked for this, then we can confirm.

**Comment by deepika-Gopala** [comment]
Design has been changed for the above widget and it works differently now and dint find the issue mentioned so far.

**Comment by veereshms** [comment]
Design has been changed and this issue is no longer relevant.
